rust_team_data/src/email_encryption.rs

```diff
@@ -0,0 +1,111 @@
+//! This module implements the encryption scheme used to safely include private email addresses in
+//! the team repository. It generates encrypted content that looks like this:
+//!
+//! encrypted+a371d1d65dcc697d000826e4bd59dcbfb16ad73eceb1139caaaab795737bb3a54403f0ec69f3d40c8965a5f4bc52f67c94ec538a8a2fea710cabdaaf27661f9a62b0121db7d229@rust-lang.invalid
```

Comment:
Maybe you want to put this in a code block so the docs are rendered correctly?

Reply:
Done!